packages/core/bootstrap/src/lib/ws/types.ts

```diff
@@ -0,0 +1,30 @@
+
+export interface WSConfig {
+  /** Info about the connection */
+  connectionInfo: {
+    key: string
+  }
+  /** Maximum amount of parallel connections */
+  connectionLimit: number
+  /** Number of ms if no data received this connection is considered dead */
+  connectionTTL: number
+  // TODO: exponential back-off
+  /** Maximum number of connection retries */
+  connectionRetryLimit: number
```

Comment:
Some of these are unused, for now.
Please open a GitHub issue and make a list of things that still need to be implemented.

Reply:
#467